src/formatDate.js

```diff
@@ -8,7 +8,75 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const arrFromDate = date.split(fromFormat[3]);
```

Comment:
The splitting of the date string using `fromFormat[3]` assumes that the separator is always at this position, which might not be the case. Consider using a more flexible approach to determine the separator.